packages/backend/src/projectAdapters/dbtBaseProjectAdapter.ts

```diff
@@ -171,6 +177,7 @@ export class DbtBaseProjectAdapter implements ProjectAdapter {
                     adapterType,
                     metrics,
                     this.warehouseClient,
+                    this.lightdashConfig.warehouseTimezone,
```

Comment:
I thought about making this a property of warehouseClient rather than a lightdashCOnfig, but this way the user will not have to `recreate`their project settings, and this is probably easier